src/Svg.Model/IAssetLoader.cs

```diff
@@ -10,4 +11,6 @@ namespace Svg.Model;
 public interface IAssetLoader
 {
     SKImage LoadImage(Stream stream);
+    List<(string text, float advance, SKTypeface? typeface)>
```

Comment:
Please don't do such log tuple types, just add simple class or record, it's hard to read.